tests/specs/scoped.js

```diff
@@ -592,7 +592,7 @@ test('scoped Store .off()', function (t) {
   })
 })
 
-test('when type change', function (t) {
+test.only('when type change', function (t) {
```

Comment:
Haha :+1: saw your tweet so I took a look! Also, You could probably make a linting rule that checks for `test.only`